RELEASE.md

```diff
@@ -2,22 +2,59 @@
 
 [![hackmd-github-sync-badge](https://hackmd.io/sVOAyv6LTwiQllQUctxP1w/badge)](https://hackmd.io/sVOAyv6LTwiQllQUctxP1w)
 
-## Resolve Issues
+## Versioning Scheme
+Flyte components follow [SemVer conventions](https://semver.org/). In a nutshell, that means every component has a release version that follows this format: `MAJOR.MINOR.PATCH`. This includes the aggregated milestone release described below.
```

Comment:
Just to circle back on this, as of the [0.18.2 release](https://github.com/flyteorg/flyte/blob/master/CHANGELOG/CHANGELOG-v0.18.2.md) we started publishing component-specific images tagged with the flyte release version. This will simplify the update story, since now users will be able to simply use the release version as opposed to be worried about which version to pick, while at the same time letting users be on the bleeding edge if needed.